content/docs/pulumi-cloud/cloud-rest-api/_index.md

```diff
@@ -1497,6 +1623,44 @@ EMPTY RESPONSE BODY
 
 <!-- ###################################################################### -->
 
+### Update User's Role
```

Comment:
ℹ️ moved this section to be adjacent to other `orgs/{organization}/members` APIs